plugins/modules/vmadm.py

```diff
@@ -207,6 +212,11 @@
     description:
       - Consider the provisioning complete when the VM first starts, rather than when the VM has rebooted.
     type: bool
+  owner_uuid:
+    required: false
+    description:
+      - This parameter can be used for defining the UUID of an ´owner´ for this VM.
```

Comment:
What kind of formatting should 'owner' have? Is this a variable, a vmadm term, ...? Backticks are not supported in Ansible markup.

Reply:
It _looks_ like the owner is always going to be one, singular, user, and also no need to _explain_  that a parameter is a parameter and can be used for something, rather we can just state what it does. So maybe we could write it as:
```suggestion
      - Define the UUID of the owner of the VM.
```

Reply:
Is this change necessary for what the Ansible community wants? I also copied this language out of the vmadm man page. 
https://smartos.org/man/8/vmadm

Here is the exact contents:

```
           This parameter can be used for defining the UUID of an 'owner' for this
           VM. It serves no functional purpose inside the system itself, but can
           be used to tie this system to others.

           type: string (UUID)
           vmtype: ANY
           listable: yes
           create: yes
           update: yes
           default: 00000000-0000-0000-0000-000000000000
```

The owner is a single UUID. I wouldn't mind modifying the comment to what @russoz suggested if that is what is preferred.

Reply:
It can make sense to take over wording from the original product, but in this case it doens't make sense. First we prefer the term 'option' over 'parameter', and second, as @russoz said, this is the documentation for an option, so it doesn't have to repeat that it's an option (parameter).